.github/workflows/test.yml

```
@@ -44,6 +44,14 @@ jobs:
         with:
           dockerfile: Dockerfile
 
+      - name: staticcheck
+        uses: dominikh/staticcheck-action@v1
+        if: ${{ !cancelled() }}
+        with:
+          version: latest
```

```suggestion
          version: v0.5.1
```
I prefer to avoid using the `latest` tag even if it means updating manually :smile: